Libraries/DirectMLX.h

```diff
@@ -210,10 +214,11 @@ namespace dml
 
     using absl::make_unique;
 #else
+    #ifndef DMLX_OPTIONAL_EXTENDED
     template <typename T>
```

Comment:
Indentation doesn't match the code right below it.

Reply:
Fixed